src/index.js

```diff
@@ -36,7 +36,9 @@ ReactDOM.render(
       <Router history={history}>
         <Route path="/auth"  component={Auth} />
         <Route path="/" component={App} />
+        {/* Leave this for legacy support */}
         <Route path="/(:username)/snippets/(:id)" component={App} />
```

Comment:
Could we include a [`<Redirect />`](https://github.com/ReactTraining/react-router/blob/v3/docs/guides/RouteConfiguration.md) component here to route them to the correct URL?
```jsx
<Route
  path="/(:username)/snippets/(:id)"
  render={<Redirect from="/(:username)/snippets/(:id)" to="/(:username)/(:id)" />}
/>
```